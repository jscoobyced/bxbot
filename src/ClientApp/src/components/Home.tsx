import * as React from 'react';

export class Home extends React.Component {
    public render() {
        return <article>
            <h1>Hello, world!</h1>
        </article>;
    }
}
