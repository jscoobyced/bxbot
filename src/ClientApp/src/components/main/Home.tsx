import * as React from 'react';
import { HomeContent } from '../../services/Models';

export interface HomeProps {
    homeContent: HomeContent;
}

export class Home extends React.Component<HomeProps> {

    public constructor(props: HomeProps) {
        super(props);
    }

    public render() {
        return <article>
            <h1>{this.props.homeContent.articleTitle}</h1>
            <section>{this.props.homeContent.article}</section>
        </article>;
    }
}
