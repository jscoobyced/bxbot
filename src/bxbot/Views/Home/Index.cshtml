@{
    ViewData["Title"] = "Home Page";
}

@section styles {
    <link rel="stylesheet" href="~/dist/style.6e45d99ee3a9c6a2d74d.css?1c5ceb03f16395a5c123">
}
<div id="app-root">Something awesome is coming...</div>

@section scripts {
    <script src="~/dist/vendor.584d115ceaf2f0e28503.js?1c5ceb03f16395a5c123"></script>
    <script src="~/dist/main.27dec449c37ccfcfb1fa.js?1c5ceb03f16395a5c123"></script>
}
