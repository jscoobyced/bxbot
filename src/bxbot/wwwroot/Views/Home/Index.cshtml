@{
    ViewData["Title"] = "Home Page";
}

@section styles {
    <link rel="stylesheet" href="~/dist/style.6e45d99ee3a9c6a2d74d.css?9708653bc96ba89fdd59">
}
<div id="app-root">Something awesome is coming...</div>

@section scripts {
    <script src="~/dist/vendor.584d115ceaf2f0e28503.js?9708653bc96ba89fdd59"></script>
    <script src="~/dist/main.2baf3179b283f2bc2311.js?9708653bc96ba89fdd59"></script>
}
