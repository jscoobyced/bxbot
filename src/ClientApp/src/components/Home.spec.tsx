import { shallow } from 'enzyme';
import * as React from 'react';
import { Home } from './Home';

test('Home component is unchanged.', () => {
    const home = shallow(<Home />);

    expect(home).toMatchSnapshot();
});