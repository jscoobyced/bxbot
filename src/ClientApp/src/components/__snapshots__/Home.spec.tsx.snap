// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component is unchanged. 1`] = `
<article>
  <h1>
    Hello, world!
  </h1>
</article>
`;
